docs/testing.rst

```diff
@@ -32,3 +38,22 @@ Once the Makefile has been run, you can run ``make test`` to run the test suite.
      mv autopush/tests/test_logging.py{.hold,}
 
 This script will cause the integration and logging tests to not run.
+
+.. _test-with-firefox:
+
+Firefox Testing
+===============
+
+To test a locally running Autopush with Firefox, you will need to edit
+several config variables in Firefox.
+
+1. Open a New Tab.
+2. Go to ``about:config`` in the Location bar and hit Enter, accept the disclaimer
+   if it's shown.
+3. Search for ``dom.push.serverURL``, make a note of the existing value (usually
```

Comment:
Protip: right-clicking the pref and choosing "Reset" will also change it back to the default. ✨